core/src/components/button/button.ionic.scss

```diff
@@ -1,32 +1,55 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
-@import "./button";
-@import "./button.ionic.vars";
 
 // Ionic Button
 // -------------------------------------------------------------------------------
-
 :host {
+  --overflow: hidden;
```

Comment:
Once we revert the import removal, then these duplicate variables should be removed as well.